glossary/comet.md

```diff
@@ -6,9 +6,20 @@ template = "doc.html"
 category = "arvo"
 +++
 
-A **comet** is a kind of [ship](../ship) on the [Arvo](../arvo) network. They are disposable identities that will not be initially trusted by others on the network. One use might be to run bots as comets.
+A **comet** is a kind of [ship](../ship) on the [Arvo](../arvo) network. They
+are free identities for which we encourage new users to boot to try things out
```

Comment:
I strongly object to referring to comets this way. They are only "free" because they are not on the azimuth and they have many uses beyond just testing if you like urbit enough to buy a planet. They aren't intended to be a trial mode or anything like that.